### DISCUSSION TITLE
请问0.17.4如何升级0.18.0

### DISCUSSION THREAD

**Discussion by clhome** [midoks/mdserver-web#665] [Q&A]
请问0.17.4如何升级0.18.0，要完全重装？之前的数据还会在吗

**Comment by midoks** [comment]
最好完全重装的。我没有写迁移程序。

如果非要在原来地址升级也行，需要记住站点信息，重复手动添加即可。其他如果mysql等等数据不影响的。

**Reply by gaoyuoppa** [reply]
> 最好完全重装的。我没有写迁移程序。
> 
> 如果非要在原来地址升级也行，需要记住站点信息，重复手动添加即可。其他如果mysql等等数据不影响的。

请问 我理解的重装是  重装系统然后安装全新的0.18.0. 
或者 通过卸载脚本卸载0.17.4后再安装0.18.0呢

**Reply by midoks** [reply]
不用重装系统， rm -rf /www/server/mdserver-web。删除面板代码即可

**Reply by CK-Tex** [reply]
> 不用重装系统， rm -rf /www/server/mdserver-web。删除面板代码即可

您好，我通过指令删除面板之后，使用update.sh升级到了0.18，之后发现网站列表空了，但是网站可以正常访问。
有办法恢复吗网站列表吗？还是需要手动一个一个添加回去？

**Comment by midoks** [comment]
mw 29   恢复站点信息
